src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const tableHeadRow = document.querySelector('thead');
+const tableHeadContent = [];
+
+const tableBody = document.querySelector('tbody');
+const tableRows = tableBody.querySelectorAll('tr');
+
+const employees = [];
+
+for (const element of tableHeadRow.querySelectorAll('th')) {
+  tableHeadContent.push(element.innerText);
+}
+
+for (const element of tableRows) {
+  const row = element.querySelectorAll('td');
+  const rowContent = [];
+  const employee = {};
+
+  for (const item of row) {
+    rowContent.push(item.innerText);
+  }
```

Comment:
you do extra unnecessary actions. Don't try to create an array with inner texts, just use them from the initial array